### PR TITLE
feat(brand): Converted brand component to typescript

### PR DESCRIPTION
**Pull request by jessiehuff**
Fixes issue #1985

### CODE REVIEW

**Comment by nicolethoen** [comment]
@jessiehuff can you also add integration tests for this component as outlined on this [README](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration)?

File: packages/patternfly-4/react-core/src/components/Brand/Brand.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export interface BrandProps extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
+  className?: string;
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Brand/Brand.tsx]: Missing description of props for documentation.

**Comment by tlabaj** [comment]
@jessiehuff could you also please add the relevant issue to the description. Thanks!

File: packages/patternfly-4/react-core/src/components/Brand/Brand.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  /** Attribute that specifies the alt text of the image for the Brand. */
+  alt: string;
+}
+
+export const Brand: React.FunctionComponent<BrandProps> = ({
+  className = '', 
+  src = '', 
+  alt,
+  ...props
+}) => (
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Brand/Brand.tsx]: Change to `}: BrandProps) => (`

**Pull request merged**